pkgs/tools/misc/urjtag/default.nix

```diff
@@ -1,31 +1,47 @@
 { stdenv, autoconf, automake, pkgconfig, gettext, intltool, libtool, bison
-, flex, fetchgit, makeWrapper
+, flex, which, subversion, fetchsvn, makeWrapper
+, svfSupport ? null
+, bsdlSupport ? null
+, staplSupport ? null
 , jedecSupport ? false
 , pythonBindings ? false
 , python3 ? null
+, libftdiSupport ? libftdi
```

Comment:
Yes, like @vcunat points out, *Support arguments are generally booleans. Having them default to a package is weird/wrong.